packages/indexer/lib/index.js

```diff
@@ -2,7 +2,7 @@ const { validators, normalizers, Reader, RPC } = require("ckb-js-toolkit");
 const { OrderedSet } = require("immutable");
 const XXHash = require("xxhash");
 const { Indexer: NativeIndexer } = require("../native");
-
+const lodash = require("lodash");
```

Comment:
Where is the `lodash` from? I can't find in `packcage.json`.

Reply:
Oops, it's in the workspace's `yarn.lock` file, I gonna add it in the package.json.